src/server/pfs/server/driver_file.go

```diff
@@ -23,6 +23,7 @@ import (
 	log "github.com/sirupsen/logrus"
 )
 
+// TODO(acohen4): signature should accept a branch seperate from the commit
```

Comment:
Undone TODO

Reply:
Is this referring to the broader API change we would eventually want to make?